### PR TITLE
Yg/tag from bdd to graph

### PR DESCRIPTION
**Pull request by yoagauthier**

### CODE REVIEW

File: src/components/graph/screens/GraphActivityDisplay/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            centerNodeSubType: res2.rows._array[0].type,
+          });
+        });
+    });
+}
+
+export function getAdjacentInterlocutorsFromActivity(activityId) {
+  return dispatch => executeSql(
+    `SELECT name, interlocutor.id as id, activity.id as activity_id
+    FROM activity INNER JOIN interlocutor ON activity.interlocutor_id = interlocutor.id
```
Comment (T0nio) [inline src/components/graph/screens/GraphActivityDisplay/actions/index.js]: Line Break for INNER JOIN ?

File: src/components/graph/screens/GraphActivityDisplay/containers/GraphActivityDisplayContainer.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import { Dimensions } from 'react-native';
+import { connect } from 'react-redux';
 import * as shape from 'd3-shape';
 import * as hierarchy from 'd3-hierarchy';
 import * as force from 'd3-force';
 import * as zoom from 'd3-zoom';
+import { GraphNodeType, ActivityTypeEnum } from 'yasav/src/const';
+import COLORS from 'yasav/src/styles/Colors';
```
Comment (T0nio) [inline src/components/graph/screens/GraphActivityDisplay/containers/GraphActivityDisplayContainer.js]: Were you angry ? Why upcase ?

File: src/components/graph/screens/GraphActivityDisplay/containers/GraphActivityDisplayContainer.js [hunk 3]
```diff
@@ -19,46 +23,32 @@ class GraphActivityDisplayContainer extends React.Component {
...
+    this.props.getAdjacentNodes(this.props.centerNodeId, GraphNodeType.TAG)
+      .then(() => {
+        this.data = this.props.nodesList;
+        this.links = this.props.linksList;
+        this.force = d3.force.forceSimulation(this.data)
+          .force('charge', d3.force.forceManyBody(-300).distanceMax(this.height / 2))
+          .force('link', d3.force.forceLink(this.links).distance(200))
+          // .force('radial', d3.force.forceRadial(500, this.width / 2, this.height / 2))
+          // .force('center', d3.force.forceCenter(this.width / 2, this.height / 2))
+          // .force('x', d3.force.forceX().strength(0.2))
```
Comment (T0nio) [inline src/components/graph/screens/GraphActivityDisplay/containers/GraphActivityDisplayContainer.js:47]: Do you still need those comments ?

Comment (yoagauthier) [reply src/components/graph/screens/GraphActivityDisplay/containers/GraphActivityDisplayContainer.js:47]: I think I will let them, then can be quite usefull if we want to try very fast new forces to make nodes intract

**Pull request merged**